<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QWidget" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>20</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Qt SQL Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="WorktimeManager_v1-35.qrc">
    <normaloff>:/images/database.png</normaloff>:/images/database.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QSplitter::handle {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
stop:0 rgba(255, 255, 255, 0), 
stop:0.407273 rgba(200, 200, 200, 255), 
stop:0.4825 rgba(101, 104, 113, 235), 
stop:0.6 rgba(255, 255, 255, 0));
    image: url(:/images/splitter.png);
     }
QTreeView {
    show-decoration-selected: 1;
}
QTreeView::item {
     border: 1px solid #d9d9d9;
    border-top-color: transparent;
    border-bottom-color: transparent;
}
QTreeView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
}
QTreeView::item:selected {
    border: 1px solid #567dbc;
}
QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}
QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}
QTreeView{
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
   border-radius: 5px;
   margin-top: 1ex; /* leave space at the top for the title */
   font: italic 9pt &quot;Arial&quot;;
   /*font-weight: bold;*/
   color: black;
}
QToolTip {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 400;
}
QLabel{
color: ;
	color: rgb(70, 70, 70);
}
/*
QTreeView::title {
   subcontrol-origin: margin; 
   subcontrol-position: top center; 
   top: 1.2ex;   
   padding: 0px 8px
} 


QGroupBox{
   border: 0.24ex dashed rgb(55, 55, 55); 
}*/
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="TabView" name="tva">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>15</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="TabView" name="tvb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>15</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="TabView" name="tvc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>15</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="ConnectionWidget" name="connectionWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>15</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>373</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
        <widget class="TabView" name="tvd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="TabView" name="tvl1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="TabView" name="tvl2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="statusTip">
    <string>Inserts a new Row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current Row</string>
   </property>
  </action>
  <action name="fieldStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Field Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Field Change</string>
   </property>
  </action>
  <action name="rowStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit on &amp;Row Change</string>
   </property>
   <property name="toolTip">
    <string>Commit on Row Change</string>
   </property>
  </action>
  <action name="manualStrategyAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submit &amp;Manually</string>
   </property>
   <property name="toolTip">
    <string>Commit Manually</string>
   </property>
  </action>
  <action name="submitAction">
   <property name="text">
    <string>&amp;Submit All</string>
   </property>
   <property name="toolTip">
    <string>Submit Changes</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="text">
    <string>&amp;Revert All</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="text">
    <string>S&amp;elect</string>
   </property>
   <property name="toolTip">
    <string>Refresh Data from Database</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabView</class>
   <extends>QTableView</extends>
   <header>tabview.h</header>
  </customwidget>
  <customwidget>
   <class>ConnectionWidget</class>
   <extends>QTreeView</extends>
   <header>connectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WorktimeManager_v1-35.qrc"/>
 </resources>
 <connections/>
</ui>
